src/components/apps/details/Drawer.js

```diff
@@ -61,10 +61,10 @@ const useStyles = makeStyles((theme) => ({
         [theme.breakpoints.up("lg")]: {
             maxWidth: "25%",
         },
-        [theme.breakpoints.down("lg")]: {
+        [theme.breakpoints.down("xl")]: {
             maxWidth: "50%",
         },
-        [theme.breakpoints.down("sm")]: {
+        [theme.breakpoints.down("md")]: {
```

Comment:
Pretty sure this was previously a bug, but reverting these code-mod changes will fix the bug in this PR.